xml/System.Text.Json.Serialization/JsonNumberHandling.xml

```diff
@@ -109,7 +109,7 @@
       <MemberValue>2</MemberValue>
       <Docs>
         <summary>Numbers will be written as JSON strings (with quotes), not as JSON numbers.</summary>
-        <remarks>To be added.</remarks>
+        <remarks>Mentioned values are not defined by the specification. Using this setting may potentially produce JSON which may produce JSON which cannot be parsed by other JSON implementations.</remarks>
```

Comment:
Why is this remark here and not in the other fields too? But I think it's fine just to have it in the main remarks for the enumeration.

Reply:
The other fields are not included in spec. 
See https://github.com/dotnet/runtime/issues/48926#issuecomment-973302102

@layomia Maybe you can say something about it :D